src/nft/components/bag/BagFooter.tsx

```diff
@@ -207,6 +207,10 @@ export const BagFooter = ({ totalEthPrice, bagStatus, fetchAssets, eventProperti
   )
 
   const usdcValue = useStablecoinValue(inputCurrency ? swapTrade?.inputAmount : parsedAmount)
+  const traceEventProperties = {
+    usd_value: usdcValue?.toExact(),
+    ...eventProperties,
```

Comment:
The past in eventsProperty is instantiated in `Bag` but only ever passed down. To reduce `BagFooter` args you could instantiate here and include the usd_value

Reply:
actually it still needs to be passed down because the other event properties come from the items in the bag that aren't being passed to the footer